tests/diff-ru/out.side-by-side

```diff
@@ -15,7 +15,7 @@
 [0m[33m--- a/common/foo.txt	2013-02-22 20:28:32.000000000 +0800
 [0m[33m+++ b/common/foo.txt	2013-02-22 20:30:18.000000000 +0800
 [0m[1;34m@@ -1 +1 @@
-[0m[33m1[0m [31m[7m[31mHello[0m[31m[0m                                                                            [0m[33m1[0m [32m[7m[32mworld[0m[32m[0m
+[0m[33m1[0m [31m[7m[31mHello[0m                                                                            [0m[33m1[0m [32m[7m[32mworld[0m
 [36mOnly in b: date.txt
 [0m[33m[0m[33m[0m[36mOnly in a: time.txt
-[0m[33m[0m[33m[0m
+[0m[33m[0m[33m[0m
```

Comment:
I noticed a regression here: EOL at line end disappeared. If input file has the char, output should keep it too.